tests/main/mount-ns/task.yaml

```diff
@@ -127,7 +127,9 @@ prepare: |
     snap connect test-snapd-mountinfo-core16:mount-observe
     snap connect test-snapd-mountinfo-core18:mount-observe
 
-    echo "Collect mountinfo from the host"
+    echo "Collect mountinfo from the host before the test"
+    echo "but make sure that persistent mount namespaces don't clobber the output"
```

Comment:
I think you can drop this line or make it a comment

Reply:
Updated in https://github.com/snapcore/snapd/pull/7668